docs/oauth.md

```diff
@@ -274,7 +275,11 @@ Tableau Desktop uses a shared client ID and client secret which is embedded in t
 
 If the authorization server supports dynamic port ranges for native applications then that should be used. This is the default behavior for Tableau OAuth clients. For example `http://localhost:*/Callback`.
 
-If the authorization server does not support this, then enable OAUTH_CAP_FIXED_PORT_IN_CALLBACK_URL, and specify a set of `<redirectUrisDesktop>` in the OAuth config. The redirectUrisDesktop must be of the form `http://localhost:[portnumber]/Callback`. If you need to use loopback address, you can enable OAUTH_CAP_SUPPORTS_HTTP_SCHEME_LOOPBACK_REDIRECT_URLS, but the `<redirectUrisDesktop>` should still specify localhost. The callback at runtime will be rewritten to something like http://127.0.0.1:55555/Callback.
+If the authorization server does not support this, then enable OAUTH_CAP_FIXED_PORT_IN_CALLBACK_URL, and specify a set of `<redirectUrisDesktop>` in the OAuth config. 
+
+### redirectUrisDesktop Format 
+- Before Tableau 2023.1, the redirectUrisDesktop must be of the form `http://localhost:[portnumber]/Callback`. If you need to use loopback address other than localhost, you can enable OAUTH_CAP_SUPPORTS_HTTP_SCHEME_LOOPBACK_REDIRECT_URLS, but the `<redirectUrisDesktop>` should still specify localhost. The callback at runtime will be rewritten to something like http://127.0.0.1:55555/Callback.
```

Comment:
Will this be a breaking change for any connectors?

Reply:
No it won't, existing plugin connectors should still work with this change, cuz the change is just loosing the constraints of redirectUri, old format of redirectUri for existing connectors still works. 

MR: https://gitlab.tableausoftware.com/connectivity/oauth-service-impl/-/commit/fd6e2e970270b8746dbb66e955929aabfb2cb045